src/api/app/assets/javascripts/webui/tabs.js

```diff
@@ -31,7 +31,7 @@ $.fn.hasOverflow = function() {
   return (element.offsetHeight < element.scrollHeight - 1) || (element.offsetWidth < element.scrollWidth - 1);
 };
 
-$.fn.collapse = function(){
+$.fn.collapseCollapsible = function(){
```

Comment:
I've renamed the function as it overrides `collapse` Bootstrap's function. Thanks @eduardoj for pointing me out:)